src/routes/index.tsx

```diff
@@ -15,6 +22,62 @@ const AppRoutes = () => (
                     </ConnectionRequired>
                 }
             />
+            <Route
+                path='usernew.html'
+                element={
+                    <ConnectionRequired>
```

Comment:
Admin routes should set the `isAdminRequired` prop on the `ConnectionRequired` component

Reply:
[28d45df](https://github.com/jellyfin/jellyfin-web/pull/3733/commits/28d45df3cc629352523756a6f28832824f81753b)